php-80/Dockerfile

```diff
@@ -384,6 +385,10 @@ ARG VERSION_PHP
 RUN curl --location --silent --show-error --fail https://www.php.net/get/php-${VERSION_PHP}.tar.gz/from/this/mirror \
   | tar xzC . --strip-components=1
 
+COPY layers/openssl3.patch ${PHP_BUILD_DIR}
+RUN patch -N -p1 -s < openssl3.patch
+RUN rm openssl3.patch
```

Comment:
As mentioned on Slack can you remove the patching for PHP 8.0?

My reasoning is:

- 8.0 is no longer maintained (even security fixes)
- this complicates the build

I'd rather keep things simple and keep 8.0 mostly for BC.

If you don't have time for this let me know, I can create a new PR based on this one.